src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/UpdateSubmission.java

```diff
@@ -28,6 +29,9 @@ public class UpdateSubmission {
     public static final String ADDITIONS_FIELD = "additions";
     public static final String CESSATIONS_FIELD = "cessations";
     public static final String TRUSTS_ADDITIONS_FIELD = "trusts";
+    public static final String CHANGE_BO_RELEVANT_PERIOD = "changeBORelevantPeriod";
```

Comment:
these constant names should end _FIELD to match the other field names

Reply:
done